cucumber/api/features/authenticate.feature

```diff
@@ -21,6 +21,20 @@ Feature: Exchange a role's API key for a signed authentication token
       cucumber:user:alice successfully authenticated with authenticator authn
     """
 
+  Scenario: Authenticate response should be encoded if asked
+    When I can POST "/authn/cucumber/alice/authenticate?encoded=true" with plain text body ":cucumber:user:alice_api_key"
```

Comment:
should be `When I POST`. we use `can` only in `Then`.

Reply:
fixed